apps/wallet-dashboard/components/Dialogs/Staking/views/EnterTimelockedAmountView.tsx

```diff
@@ -32,32 +50,53 @@ export interface FormValues {
 interface EnterTimelockedAmountViewProps {
     selectedValidator: string;
     maxStakableTimelockedAmount: bigint;
+    amountWithoutDecimals: bigint;
+    senderAddress: string;
     onBack: () => void;
-    onStake: () => void;
-    gasBudget?: string | number | null;
     handleClose: () => void;
-    hasGroupedTimelockObjects?: boolean;
-    isTransactionLoading?: boolean;
+    onSuccess: (digest: string) => void;
 }
 
 function EnterTimelockedAmountView({
-    selectedValidator: selectedValidatorAddress,
+    selectedValidator,
     maxStakableTimelockedAmount,
-    hasGroupedTimelockObjects,
+    amountWithoutDecimals,
+    senderAddress,
     onBack,
-    onStake,
-    gasBudget,
     handleClose,
-    isTransactionLoading,
+    onSuccess,
 }: EnterTimelockedAmountViewProps): JSX.Element {
     const account = useCurrentAccount();
     const accountAddress = account?.address;
```

Comment:
we can remove the `accountAddress` because we have it already in the props as `senderAddress`